app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,23 @@
   </head>
 
   <body>
+    <div>
+      <nav>
+        <br>
+        <% if !user_signed_in?%>
+            <%= link_to 'Sign Up',  new_user_registration_path  %>
+            <%= link_to 'Sign In',  new_user_session_path  %>
+            <% end%>
+        <br>
+        <% if user_signed_in?%>
+            <%= link_to 'Sign Out', destroy_user_session_path, method: :delete %>
+            <%= link_to 'Edit Profile',  edit_user_registration_path %>
+        <% end%>
```

Comment:
Use else instead of rechecking the variable

Reply:
Got it.